File: 1010/1010.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");
var lines = input.split("\n");

var [cod1, num1, valor1] = lines[0].split(" ").map(Number);

var [cod2, num2, valor2] = lines[1].split(" ").map(Number);

var total = num1 * valor1 + num2 * valor2;

console.log("VALOR A PAGAR: R$ " + total.toFixed(2));

File: 1010/1010.py
linha1 = input().split()
linha2 = input().split()

cod1 = int(linha1[0])
num1 = int(linha1[1])
valor1 = float(linha1[2])
cod2 = int(linha2[0])
num2 = int(linha2[1])
valor2 = float(linha2[2])

total = ((valor1 * num1) + (valor2 * num2))

print(f"VALOR A PAGAR: R$ {total:.2f}")
